overlord/fdestate/backend/reseal.go

```diff
@@ -61,6 +61,7 @@ func MockSecbootBuildPCRProtectionProfile(f func(modelParams []*secboot.SealKeyM
 }
 
 type StateUpdater func(role string, containerRole string, bootModes []string, models []secboot.ModelForSealing, tpmPCRProfile []byte) error
+type StateGetter func(role string, containerRole string) (bootModes []string, models []secboot.ModelForSealing, tpmPCRProfile []byte, err error)
```

Comment:
wouldn't it make sense to combine these two into an interface and have FDEManager implement it?

Reply:
Yes. I will do that.

Reply:
Though, I will need to wrap it since we need to handle the unlocker.